microsoft/testsuites/power/power.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.features import Hibernation, Sriov, StartStop, Synthetic
+from lisa.nic import Nics
+from lisa.testsuite import simple_requirement
+from lisa.tools import HibernationSetup
+from lisa.util import LisaException
+from lisa.util.perf_timer import create_timer
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="power",
+    category="functional",
+    description="""
+        This test suite is to test hibernation in guest VM.
+    """,
+)
+class Power(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            to do
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            network_interface=Synthetic(),
+            supported_features=[Hibernation],
+        ),
+    )
+    def verify_hibernation_synthetic_network(self, node: Node, log: Logger) -> None:
+        self._verify_hibernation(node, log)
+
+    @TestCaseMetadata(
+        description="""
+            to do
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            min_nic_count=3,
```

Comment:
Why a hibernation test case needs three nics?

Reply:
hmm, last time there is a bug when enable sriov, after resume, one vf changed into two, I just want to make sure there is no VF count decrease or increase, just my own thoughts.

Reply:
now I changed into one nic.

Reply:
Good. If you thought there is a bug, and it needs a regression. Please create a separated test case and specify in the description.